.claude/rules/testing.md

```
@@ -0,0 +1,99 @@
+it('has no accessibility violations', async () => {
+    const { container } = render(<Button>Click me</Button>);
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+});
+```
+
+## Colocation
+
+Keep test files next to the component they cover:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)